force-app/main/educationCloudSettings/lwc/edcSettingsProductCard/edcSettingsProductCard.js

```diff
@@ -68,4 +78,55 @@ export default class EdcSettingsCard extends NavigationMixin(LightningElement) {
         };
         this[NavigationMixin.Navigate](pageReference);
     }
+
+    get showProductCard() {
+        if (this.isDisplayProduct && 
```

Comment:
Consider a different name that describes what this is doing. Something like displayProductCards that makes it abundantly clear what this variable's purpose is.

Reply:
Done. Thanks